test/cmdlineTests/model_checker_invariants_contract_reentrancy/err

```diff
@@ -1,14 +1,8 @@
-Warning: Return value of low-level calls not used.
- --> model_checker_invariants_contract_reentrancy/input.sol:6:3:
-  |
-6 | 		_a.call("");
-  | 		^^^^^^^^^^^
-
 Info: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
 
 Info: Contract invariant(s) for model_checker_invariants_contract_reentrancy/input.sol:test:
-(!(x >= 1) || true)
+(true || !(x >= 1))
 Reentrancy property(ies) for model_checker_invariants_contract_reentrancy/input.sol:test:
-(((!(x <= 0) || !(<errorCode> >= 1)) && (!(x <= 0) || !(x' >= 1))) || true)
+(((<errorCode> = 0) && (x!6 = x!4) && (x' = x)) || true || true || true || true)
```

Comment:
What does `x!6 = x!4` mean?

Reply:
Looks like these are variables names in Z3 that we are for some reason not translating back (to expressions like `x` and `x'`).
That is something to look into, but in general the invariants from Z3 are not very informative at the moment.
It basically always contains some expression that we are not able to translate back to Solidity language, so we currently overapproximate such parts with `true`. But that makes the invariant trivial.
See also https://github.com/ethereum/solidity/issues/13937